ui/api/index.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
```

Comment:
@aabidsofi19 make sure that you highlight this sort of pervasive change prominently in the PR description.

What is this for and what does it do?

Reply:
sorry , forgot we were not using axios in meshery ui  , although it was in dependencies  . Btw this is the axios instance we use for data fetching (in  meshmap also in meshery-cloud ) but seems not in meshery ui . should i remove it .  @leecalcote